src/__integrations__/pmClient.startSession.integration.test.ts

```diff
@@ -41,41 +41,13 @@ describe('Endpoint startSession', () => {
     await pmServerTerminator.terminate();
   });
 
-  it('should return 500 when the request is empty', async () => {
-    expect(
-      (
-        await pmClient.startSessionUsingPOST({
-          startSessionRequest: {},
-        })
-      ).fold(
-        err => (err.pop()?.value as Response).status,
-        myRes => myRes.status,
-      ),
-    ).toEqual(500);
-  });
-
-  it('should return 422 when the request has an empty data field', async () => {
-    expect(
-      (
-        await pmClient.startSessionUsingPOST({
-          startSessionRequest: {
-            data: {},
-          },
-        })
-      ).fold(
-        err => (err.pop()?.value as Response).status,
-        myRes => myRes.status,
-      ),
-    ).toEqual(422);
-  });
```

Comment:
why do you remove test KO response `500 Internal Server Error` with empty empty `{}` data request and `422 Unprocessable Entity` with partial empty data (ex. without `email`) ?

Reply:
@pasqualespica After fixing the spec, I can't have a void data or a data without email otherwise the compiler will complain